examples/src/bin/lotr/main.rs

```diff
@@ -45,8 +45,11 @@ fn main() {
     } else {
         let g = GraphDB::new(2);
         let now = Instant::now();
-
+        let delimiter_string = ",";
+        let delimiter: u8 = delimiter_string.as_bytes()[0];
         let _ = CsvLoader::new(data_dir)
+            .set_header(true)
+            .set_delimiter(delimiter)
```

Comment:
Create an api `set_delimiter_str` that takes `delimiter_string` directly and creates `delimiter` and passes it to `set_delimiter` on our behalf